homeassistant/components/group/light.py

```diff
@@ -82,6 +82,23 @@ async def async_setup_platform(
     )
 
 
+FORWARDED_ATTRIBUTES = frozenset(
```

Comment:
frozensets are unordered. Make it a tuple instead

Reply:
The frozenset is used for lookups, not for iteration. Making it a tuple would be much more inefficient as it would require a sequential lookup for each attribute (equivalent to using a list of pairs instead of using a dictionary):

```python
data = {
            key: value for key, value in kwargs.items() if key in FORWARDED_ATTRIBUTES
       }
```

The iteration is done over `kwargs`.

Reply:
Ah right, we don't iterate over this list, just do lookups. This is good 👍